includes/definitions/patton-sn.yaml

```diff
@@ -0,0 +1,14 @@
+os: patton-sn
+group: patton
+text: 'Patton SmartNode'
+type: network
+icon: patton
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'Processor Usage' }
```

Comment:
Maybe overs could be proc, temp and states(not sure about the last one)?

Reply:
Mempools is here now. Had forgot about it. Device traffic is indeed useless on these. Count of calls makes more sense.